docs/sources/alerting/unified-alerting/fundamentals/alertmanager.md

```diff
@@ -15,3 +15,24 @@ Grafana includes built-in support for Prometheus Alertmanager. By default, notif
 Grafana 8 alerting added support for external Alertmanager configuration. When you add an [Alertmanager data source]({{< relref "../../../datasources/alertmanager.md" >}}), the Alertmanager drop-down shows a list of available external Alertmanager data sources. Select a data source to create and manage alerting for standalone Cortex or Loki data sources.
 
 {{< figure max-width="40%" src="/static/img/docs/alerting/unified/contact-points-select-am-8-0.gif" max-width="250px" caption="Select Alertmanager" >}}
+
+## External Alertmanagers
+
+It is possible to add one or many external alertmanagers to your alertmanager configuration. This will route your alerts to this/these alertmanager(s) as well as the internal one (??). This is done via the admin tab in the alerting ui.
+
+### Add a new external Alertmanager
+
+1. In the Grafana menu, click the Alerting (bell) icon to open the Alerting page listing existing alerts.
+2. Click Admin
+3. Scroll down to the External Alertmanager section
+4. Click the Add Alertmanager button, a modal opens.
+5. Add the url and eventual port to the alertmanager, there is no need to specify`/api/v(1|2)/alerts`. Grafana will add this.
+6. The url will be shown in the table with a pending status, after a short while when Grafana has verified/connected/resolved it will change to active.
```

Comment:
```suggestion
6. The URL is shown in the table with a pending status, after a short period Grafana will verify that the Alertmanager is discovered and it will change to active. No requests are made to the external Alertmanager at this point, the verification is meant to signal that we're ready to send but not that sends will be successful. 
```